### DISCUSSION TITLE
Hide cards below the follow notifications on the Explore page

### DISCUSSION THREAD

**Discussion by typed-sigterm** [community/community#112443] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

It seems unnecessary to introduce me to myself :)
![Screenshot_20240314_193505_com github android](https://github.com/community/community/assets/145281501/9f1c8f50-369f-4f48-861a-aeff484cc144)

**Comment by taki-on** [comment, marked as answer]
Thank you for the suggestion @Zihan-Hu . Yeah it is maybe redundant information for the viewer 🤔 Let me discuss it with our UX designers!

**Comment by typed-sigterm** [comment]
On github.com, the card shows who follows me. But on GitHub Mobile, the problem persists.
